resources/migrations/0016_auto_20151028_1653.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import autoslug.fields
+import resources.models.utils
+
+
+class Migration(migrations.Migration):
```

Comment:
Can you explain why it can't be set in the first migration?

The only thing that comes to mind is that all of the resources will have `""` as their slug before the RunPython migration, which would be a violation of that uniqueness constraint. You _could_ set `null=True, unique=True` on the slug field to work around that (as they'd then default to `NULL`, which is not a value that's considered unique).

In any case, the AlterField operations could be moved into the other migration, after the RunPython op, to have only one migration per feature.